duckduckgo/iOS
# Updated to latest bloom filter and eliminate calls to smarter encryption service

Task/Issue URL: https://app.asana.com/0/72649045549333/1195088920676363

**Description**:
- Update to latest bloom filter
- Embed smarter encryption data in app
- Remove backend calls to smarter encryption

**Steps to test this PR**:

Test fresh installation with backend data:
1.  Install a fresh installation of the app, wait for all data to sync
1. Navigate to http://facebook.com and ensure this is upgraded
1. Navigate to some long url that doesn't exist and is thus definitely not in our list e.g http://thisUrlDoesNotExistAndIsNotInOurSmarterEncryptionList. com and ensure this is not upgraded

Test fresh installation with embedded data:
1. Block all calls to out smarter encryption endpoints in Charles these are https://staticcdn.duckduckgo.com/https/https* or make breaking changes to the urls at AppUrls.swift so they definitely fail
1.  Install a fresh installation of the app
1. Navigate to http://facebook.com and ensure this is upgraded
1. Navigate to some long url that doesn't exist and is thus definitely not in our list e.g http://thisUrlDoesNotExistAndIsNotInOurSmarterEncryptionList. com and ensure this is not upgraded

Test migration:
1. Install and run app from develop, making sure that all data is downloaded
1. Now install this branch over the top
1. Ensure no db migration crashes occur
1. Navigate to http://facebook.com and ensure this is upgraded
1. Navigate to some long url that doesn't exist and is thus definitely not in our list e.g http://thisUrlDoesNotExistAndIsNotInOurSmarterEncryptionList. com and ensure this is not upgraded



**Device Testing**:


**OS Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
